functions-and-scope.js

```diff
@@ -71,12 +99,30 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // Log het antwoord in de terminal.
 
 // ---- Verwachte uitkomst: 9
+       let compNum = 0;
 
+       for (let i = 0 ; i < grades.length ; i++) {
+           if(compNum < grades[i])  {
+                   compNum = grades[i];
+                           }
+          }
```

Comment:
Let wel op je indentation, want ik zie dit ook op andere plekken terugkomen en komt de leesbaarheid niet ten goede
```suggestion
           if(compNum < grades[i])  {
               compNum = grades[i];
           }
       }
```